### DISCUSSION TITLE
How to remove duplicate test results in the allure suite?

### DISCUSSION THREAD

**Discussion by RathinaaDeepak** [allure-framework/allure2#2096] [Questions & Support] [answered]
We are rerunning failed flaky tests using a retry mechanism. If the test is flaky we execute them one more time in the same build. Allure report is displaying the results of both the runs. Is there a way to display only the results of last/recent run of the test case?

**Comment by baev** [comment]
Please specify the framework and integration you're using.

**Reply by RathinaaDeepak** [reply]
@baev 
We are using selenium with cucumber. 
Cucumber Junit runner is used to execute the tests. The rerun formatter of the cucumber runner stores the failed tests in a text file (rerun.txt) which is again re executed in the same build. When that happens allure updates both the runs (first failed run and the re tried run).

Both have the same test case name and same parameters.

All the above are steps (inital execution, retry execution and allure report generation) are done via maven as build steps in team city.

Is it possible to update the results of  the last run alone?

**Reply by baev** [reply, marked as answer]
Ensure you're using 2.24.0 (for both `allure-cucumberXjvm` & `allure-commandline`). If the problem still exists, please create a sample project (GitHub repo) to reproduce the issue. That will help a lot. Thanks.
